python/pyspark/sql/context.py

```diff
@@ -686,9 +687,10 @@ def _ssql_ctx(self):
                 self._scala_HiveContext = self._get_hive_ctx()
             return self._scala_HiveContext
         except Py4JError as e:
-            raise Exception("You must build Spark with Hive. "
-                            "Export 'SPARK_HIVE=true' and run "
-                            "build/sbt assembly", e)
+            print("You must build Spark with Hive. "
+                          "Export 'SPARK_HIVE=true' and run "
```

Comment:
This line is overindented. The leftmost quote needs to line up with the one on the previous line.